<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditDevice</class>
 <widget class="QDialog" name="AddEditDevice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="deviceNameEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="deviceAddress">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QComboBox" name="deviceTypeComboBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Device name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Device modbus address:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Device type: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>AddEditDevice</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AddEditDevice</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
